### ISSUE TITLE
Add CONTRIBUTING.md

### ISSUE THREAD

**Issue by ChadoNihi** [rtcharity/eahub.org#859]

**Comment by patbl** [comment]
I think there's a pretty low chance that someone else is working on a given unassigned issue that has no PRs. If you dibs an issue, wait a few days, and receive no response, you can be more confident no one else is working on it.

My problem hasn't been redundant work, but rather that my PRs haven't been merged or commented on.

**Comment by sebgrebe** [comment]
@ChadoNihi When someone picks up a ticket, they should put an "In progress" label on a ticket. So if there is no label, feel free to pick up a ticket. If you want to be 100% sure, you can announce in the slack channel that you've picked up the ticket.

**Comment by sebgrebe** [comment]
@patbl Sorry, we've not been doing a good job with reviewing and merging recently. We'll try to catch up on outstanding PRs soon.

**Comment by patbl** [comment]
@sebgrebe That's fine! Thanks for getting around to them.

> When someone picks up a ticket, they should put an "In progress" label on a ticket.

Maybe you know this already, but outside collaborators don't have permission to tag issues. (I wasn't sure whether you were including outside collaborators in "someone.")

**Comment by sebgrebe** [comment]
> Maybe you know this already, but outside collaborators don't have permission to tag issues. (I wasn't sure whether you were including outside collaborators in "someone.")

No, I wasn't aware of that! Will investigate whether we can do something about that

**Comment by michaltrzesimiech** [comment]
I tentatively think it's better to have it like that but make the process clearer for people who want to contribute. I'm going to update README.md with some pointers. Meanwhile, let me add you to our team, folks.

**Comment by victor-yunenko** [comment]
I guess it's what I've been slowly composing here as a google doc for now - https://docs.google.com/document/d/1v_dxO8Djg0_dJGbgD7aYrnugGEKbR84lppT35gKFafc/edit
